src/main/java/com/iota/iri/crypto/Curl.java

```diff
@@ -115,13 +86,40 @@ private void transform() {
     public void reset() {
         Arrays.fill(state, (byte) 0);
     }
-    public void reset(boolean pair) {
+
+
+    // BCURLT - pair Curl implementation.
+    // code not in use.
```

Comment:
//////////////////////////////////////////////////////////////////////////////Code not in use/////////////////////////////////////////////////////////////////////////